### DISCUSSION TITLE
CTKOptionMenu title

### DISCUSSION THREAD

**Discussion by mospina68** [TomSchimansky/CustomTkinter#2182] [Q&A]
Hello :). I am new to using customtkinter and I am working on my first project. How can I change the CTKOptionMenu visible on the interface to leave it blank without affecting the options in the menu?

![IMAGEN](https://github.com/TomSchimansky/CustomTkinter/assets/124363519/9b14c604-df91-4ea9-976d-82d836ef0a15)

```
def controles_menu_lateral(self):
        ancho_menu = 20
        alto_menu = 30

        self.archivo_menu = tkc.CTkOptionMenu(self.menu_lateral)
        self.editar_menu = tkc.CTkOptionMenu(self.menu_lateral)

        menu_info = [
            (["1", "2", "3", "4", "5"], self.archivo_menu),
            (["6", "7", "8", "9", "10"], self.editar_menu)
        ]

        for values, menu in menu_info:
            self.configurar_boton_menu(values, menu, ancho_menu, alto_menu)
            
    def configurar_boton_menu(self, values, menu, ancho_menu, alto_menu):
        menu.configure(values=values, width=ancho_menu, height=alto_menu)
        menu.pack(side=tk.TOP, pady=20, padx=30)
```

**Comment by dipeshSam** [comment]
@mospina68 If you've used `pack()` method for placing, use `pack_forget()` method to make the widget disappeared without losing any data. Keep in mind that, to restore the widget again, you need to use `pack()` method again.

Similarly, if you've used `grid()` or `place()` methods to place the widget, use `grid_forget()` or `place_forget()` methods respectively.

Hope it will be helpful for you.

**Reply by mospina68** [reply]
Thank you :)
